Packs/Rapid7_Nexpose/Playbooks/playbook-Nexpose_-_GenericPolling_-_Create_and_download_a_report.yml

```
@@ -0,0 +1,436 @@
+id: Nexpose - Create and download a report
```

see [naming conventions](https://xsoar.pan.dev/docs/playbooks/playbook-conventions#playbooks)
update the file name to match the playbook name
```suggestion
id: Nexpose - Create and Download Report
```